app/script/controllers/login.controller.js

```diff
@@ -3,10 +3,9 @@
         .controller('loginController', function ($http, apiService, $scope, $location) {
             $scope.login = function () {
                 apiService.login($scope.user.email, $scope.user.password)
-                    .then(function (response) {
-                        console.log(response);
+                    .then(function () {
                         apiService.getProducts();
-                        $location.path( "/" );
+                        $location.path( "/products" );
```

Comment:
After login send the user to the home page ( "/" ), because we don't know if they want to check the products right away :)